### DISCUSSION TITLE
Scroll-wheel support

### DISCUSSION THREAD

**Discussion by TCMine** [File-New-Project/EarTrumpet#1018] [Completed ideas]
Allow users to use the scroll-wheel on the tray icon to change the volume. (like in Winaero's SimpleSndVol)

**Comment by davuxcom** [comment]
Scrolling over the EarTrumpet icon will adjust the master volume of the current default device. It has since `2.1.2.0`.

**Comment by riverar** [comment]
Moving to completed ideas.
